framework_tvb/tvb/core/neocom/h5.py

```diff
@@ -223,8 +225,19 @@ def store_view_model(view_model, base_dir):
                     store_view_model(model_attr[idx], base_dir)
             else:
                 store_view_model(model_attr, base_dir)
+
+        if not references and isinstance(view_model, ViewModel) and not file_exist_before_save:
+            operation = dao.get_operation_by_id(os.path.basename(base_dir))
+            disk_size = get_view_model_disk_size(h5_path)
+            operation.disk_size = operation.disk_size + disk_size
+            dao.store_entity(operation)
+
     return h5_path
 
+def get_view_model_disk_size(path):
```

Comment:
There is already a similar method:` FilesHelper.compute_size_on_disk`
Maybe it could be extended to also compute the size of an entire folder?